apm-agent-core/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1 @@
+junit.jupiter.execution.parallel.enabled=true
```

Comment:
Why here? Could there be negative implications like increased flakyness? (if not - I am all for it of course 🙂 )

Reply:
I did it here just being cautious. Maybe we should discuss in the team about this capability/option?

Reply:
Definitely. I would treat that in a separate PR though, where we can isolate related test execution changes.